src/Uno.UI/UI/Xaml/Controls/Flyout/FlyoutPlacementMode.cs

```diff
@@ -1,15 +1,52 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
+﻿using Uno;
 
 namespace Windows.UI.Xaml.Controls.Primitives
 {
+	/// <summary>
+	/// Placement of the Flyout relative to its target control.
+	/// </summary>
+#if !__WASM__
+	[NotImplemented] // Currently only supported on Wasm
```

Comment:
?? FlyoutPlacementMode is used on the other platforms.

Reply:
It's a draft PR...  I wasn't sure yet it was available on other platforms...